src/index.ts

```diff
@@ -459,7 +459,6 @@ export class AzureOpenAI extends OpenAI {
         throw new Error('Expected request body to be an object');
       }
       const model = this._deployment || options.body['model'];
-      delete options.body['model'];
```

Comment:
Maybe add a simple test?

Reply:
Added in https://github.com/deyaaeldeen/openai-node/pull/4/commits/4ede3ff1908c31659f863a80277e99d050204508